src/components/ContactUs/ContactUsInformation/ContactUsInformation.tsx

```diff
@@ -0,0 +1,15 @@
+import { ContentContactUsInformation } from './ContactUsInformation.styles'
+
+import SocialMedia from 'components/SocialMedia/SocialMedia'
+import ContactUsContent from './ContactUsContent/ContactUsContent'
+
+const ContactUsInformation: React.FC = () => {
+  return (
+    <ContentContactUsInformation>
```

Comment:
"ContentContactUsInformation" 
that's not a good name, try using Wrapper to make your code easier to read